WS2012R2/lisa/remote-scripts/ica/modules_check.sh

```diff
@@ -155,20 +155,31 @@ case $DISTRO in
     ;;
 esac
 
-if [ -f /boot/initramfs-0-rescue* ]; then
-    img=/boot/initramfs-0-rescue*
-else
-  if [ -f "/boot/initrd-`uname -r`" ]; then
-    img="/boot/initrd-`uname -r`"
-  fi
+if [ "${img:-UNDEFINED}" = "UNDEFINED" ]; then
+    if [ -f /boot/initramfs-0-rescue* ]; then
+        img=/boot/initramfs-0-rescue*
+    else
+    if [ -f "/boot/initrd-`uname -r`" ]; then
+        img="/boot/initrd-`uname -r`"
+    fi
 
-  if [ -f "/boot/initramfs-`uname -r`.img" ]; then
-    img="/boot/initramfs-`uname -r`.img"
-  fi
+    if [ -f "/boot/initramfs-`uname -r`.img" ]; then
+        img="/boot/initramfs-`uname -r`.img"
+    fi
 
-  if [ -f "/boot/initrd.img-`uname -r`" ]; then
-    img="/boot/initrd.img-`uname -r`"
-  fi
+    if [ -f "/boot/initrd.img-`uname -r`" ]; then
+        img="/boot/initrd.img-`uname -r`"
+    fi
+    fi
+else
+    if [ $img == "kdump.img" ] && [ -f /boot/initramfs-`uname -r`kdump.img ]; then
```

Comment:
I don't think we need this - $img == "kdump.img"
The if for /boot/initramfs-`uname -r`kdump.img should go in line with the previous checks for different image names.
Please also remove then the new variable from xml

Reply:
Hi Chris, thanks for your review.  I add the $img variable to check the modules whether included in "initramfs-uname -rkdump.img", without it, will check /boot/initramfs-0-rescue*, so $img makes sense. If I am wrong, you can correct me, thanks!